src/contract_wrappers/modules/checkpoint/dividend_checkpoint_wrapper.ts

```diff
@@ -512,9 +512,15 @@ export default abstract class DividendCheckpointWrapper extends ModuleWrapper {
       assert.isNonZeroETHAddressHex('token', token);
       const erc20Detailed = await this.erc20DetailedContract(token);
       const erc20TokenBalance = await erc20Detailed.balanceOf.callAsync(callerAddress);
-      const erc20TokenAllowance = await erc20Detailed.allowance.callAsync(callerAddress, token);
-      assert.assert(erc20TokenAllowance.isGreaterThanOrEqualTo(amount), 'Your allowance is less than dividend amount');
-      assert.assert(erc20TokenBalance.isGreaterThanOrEqualTo(amount), 'Your balance is less than dividend amount');
+      const erc20TokenAllowance = await erc20Detailed.allowance.callAsync(callerAddress, await this.address());
+      assert.assert(
+        erc20TokenAllowance.isGreaterThanOrEqualTo(amount.absoluteValue()),
```

Comment:
Why is `isAbsoluteValue` necessary here?

Reply:
Im using it to satisfy a warning:

“Argument type BigNumber is not assignable to parameter type BigNumber.Value”

Any better approaches?

Reply:
I just pulled your branch, removed those conversions, ran `yarn build` and everything went fine. Try deleting `node_modules` and running `yarn` again, colsing your editor, re-opening it, and then try building again without the unnecessary conversions